app/view/fund_transfer.py

```diff
@@ -0,0 +1,152 @@
+from app.model.account_transaction_details import FundTransfer, TransactionType, AccountTransactionDetails
+from flask import request
+from flask_restful import Resource
+from app.common.ResponseGenerator import ResponseGenerator
+from flask_api import status
+from app.common.logging import *
+from app.model.bank_account import BankAccount
+
+
+class FundTransferInfo(Resource):
+
+    def post(self):
+
+        """Add fund transfer in the FundTransfer table"""
+        try:
+            fund_data = request.get_json()
+            result = fund_transfer_schema.validate(fund_data)
+            if result:
+                logger.exception(result)
+                response = ResponseGenerator(data={}, message=result, success=False,
+                                             status=status.HTTP_400_BAD_REQUEST)
+                return response.error_response()
+            fund = FundTransfer(from_account=fund_data['from_account'],
+                                to_account=fund_data['to_account'])
+            db.session.add(fund)
+            db.session.commit()
+            output = fund_transfer_schema.dump(fund)
+            from_account = BankAccount.query.filter(BankAccount.account_number == fund_data['from_account']).first()
+            to_account = BankAccount.query.filter(BankAccount.account_number == fund_data['to_account']).first()
+            transaction_type = TransactionType.query.filter(TransactionType.transaction_type == "debit").first()
+            if from_account.balance - fund_data['transaction_amount'] > 1000:
+                from_account.balance -= fund_data['transaction_amount']
+                db.session.add(from_account)
+                to_account.balance += fund_data['transaction_amount']
+                db.session.add(to_account)
+                transaction_status = "success"
+            else:
+                transaction_status = "fail"
+            account = AccountTransactionDetails(transaction_amount=fund_data['transaction_amount'],
+                                                bank_account_id=from_account.id,
+                                                transaction_type_id=transaction_type.id,
+                                                fund_id=output.get('id'),
+                                                transaction_status=transaction_status)
+            db.session.add(account)
+            db.session.commit()
+            account_transaction_detail_schema.dump(account)
+            logger.info("fund transfer data successfully created")
+            response = ResponseGenerator(data=output, message="fund transfer data successfully created",
+                                         success=True, status=status.HTTP_201_CREATED)
+            return response.success_response()
+        except Exception as error:
+            logger.exception(error)
+            response = ResponseGenerator(data={}, message="Missing or sending incorrect data to create an activity",
```

Comment:
Do not write message here. Use `error` instead so that we can get the actual failure reason.

Reply:
have updated the message as error in all the APIs